frontend/src/metabase/databases/components/DatabaseForm/DatabaseForm.tsx

```diff
@@ -176,16 +179,38 @@ const DatabaseFormFooter = ({
   const { values } = useFormikContext<DatabaseData>();
   const isNew = values.id == null;
 
+  const docsUrl = useSelector(state =>
+    // eslint-disable-next-line no-unconditional-metabase-links-render -- Metabase setup + admin pages only
+    getDocsUrl(state, { page: "databases/connecting" }),
+  );
+
   if (isAdvanced) {
     return (
-      <div>
-        <FormSubmitButton
-          disabled={!isDirty}
-          title={isNew ? t`Save` : t`Save changes`}
-          primary
-        />
+      <FormFooter>
+        <Flex justify="space-between" align="center" w="100%">
+          {isNew ? (
+            <ExternalLink
+              key="link"
+              href={docsUrl}
+              style={{ fontWeight: 500, fontSize: ".875rem" }}
+            >
+              {t`Need help connecting?`}
+            </ExternalLink>
+          ) : (
+            <div />
+          )}
+
+          <Flex gap="sm">
+            <Button type="button" onClick={onCancel}>{t`Cancel`}</Button>
+            <FormSubmitButton
+              disabled={!isDirty}
+              title={isNew ? t`Save` : t`Save changes`}
+              primary
+            />
+          </Flex>
+        </Flex>
         <FormErrorMessage />
-      </div>
+      </FormFooter>
```

Comment:
would love it if this puppy stuck to the bottom of the modal for us plebs with non-retina displays

![Screenshot 2025-03-07 at 3 05 50 PM](https://github.com/user-attachments/assets/2c1e8bd0-76ec-419a-bb7c-d1ea1b949bb9)

Reply:
I have a polish pass that I'm going to do as milestone 3, would you be opposed to it happening then? @kdoh do you see this as worth blocking this PR?